__tests__/Unit/Components/Tabs/Tab.test.tsx

```diff
@@ -36,11 +58,40 @@ describe('Tabs Component', () => {
                 onSelect={onSelectMock}
             />
         );
-        const assignedBtn = screen.getByRole('button', { name: /ASSIGNED/i });
+        const assignedBtn = screen.getByRole('button', { name: 'ASSIGNED' });
```

Comment:
why did we modify this?

Reply:
This test was failing with the `/i` flag, but was passing when it is removed.
This test is failing only when we test with `ASSIGNED` as active tab.
Even though the display name is `ASSIGNED` it's failing.

Reply:
The name parameter in `getByRole` is checked as a regular expression.
So if we have the `/i` flag it will run into an issue as we are having 2 buttons which have `ASSIGNED` i.e. `ASSIGNED` and `UNASSIGNED` and the function is designed to return a single element.